core/src/test/java/net/adoptopenjdk/icedteaweb/jvm/JvmUtilsTest.java

```diff
@@ -242,4 +218,35 @@ public void testMoreThanOneBlankBtwnArgs() {
             fail(ile.getMessage());
         }
     }
+
+    @Test
+    public void testSomeNewArgs() {
+        final String java_vm_args = "-XX:+UseParNewGC -XX:+CMSParallelRemarkEnabled -Dsun.net.client.defaultConnectTimeout=5000 -Dsun.net.client.defaultReadTimeout=500 -Djava.locale.providers==COMPAT,SPI";
+        try {
+            JvmUtils.checkVMArgs(java_vm_args);
+        } catch (IllegalArgumentException ile) {
+            fail(ile.getMessage());
+        }
+    }
+
+    @Before
+    public void setConfigJvmArgs() {
```

Comment:
Please move this method to the top of the class.
Otherwise this will not be noticed by other devs.